### PR TITLE
Update the changelog with old PRs and github issue ids (USDU-138)

### PR DESCRIPTION
**Pull request by judubu**
## Purpose of this PR

**Ticket/Jira #:** https://jira.unity3d.com/browse/USDU-138

The changelog needed an update to include PRs previously merged in dev.

### CODE REVIEW

File: package/com.unity.formats.usd/CHANGELOG.md [hunk 1]
```diff
@@ -1,22 +1,27 @@
...
+- Added USDZ export from recorder clip.
+- Added Scope import as XForm (#154).
 
 ### Bug Fixes
-- Fix the scene primMap that was corrupted with invalid prim making the write fail.
-- Fix an inability to refresh layers in Editor Mode
+- Fixed a crash happening when calling Scene.Read on non existing prims (#158).
+- Fixed an inability to refresh layers in Editor Mode (#155).
 - Fixed USDPayload's "Is Loaded" field in the inspector staying at false even when the payload has been loaded.
 - Fixed Transform Overrides export extension (USDU-124)
```
Comment (cguer) [inline package/com.unity.formats.usd/CHANGELOG.md:17]: I know it's not part of this PR's changes but do we want to reference internal JIRA ids in the public changelog?

**Review by cguer — COMMENTED** [review]
I didn't cross-check all user-submitted PRs but I remembered this one that changes the default export FPS from 30 to 60 https://github.com/Unity-Technologies/usd-unity-sdk/pull/170

Do you think it's worth mentioning in the changelog?

File: package/com.unity.formats.usd/CHANGELOG.md (hunk 1 above)
Comment (judubu) [reply package/com.unity.formats.usd/CHANGELOG.md:17]: Don't think so. Github issues are more useful to the user and we should have jira issue numbers in the commit messages.

**Comment by judubu** [comment]
> I didn't cross-check all user-submitted PRs but I remembered this one that changes the default export FPS from 30 to 60 #170
Done.

**Review by cguer — APPROVED** [review]
QA approved 👍!

**Pull request merged**